mobile/lib/main.dart

```diff
@@ -63,12 +63,23 @@ import 'features/stable/stable_value_change_notifier.dart';
 final GlobalKey<NavigatorState> _rootNavigatorKey = GlobalKey<NavigatorState>(debugLabel: 'root');
 final GlobalKey<NavigatorState> _shellNavigatorKey = GlobalKey<NavigatorState>(debugLabel: 'shell');
 
-void main() {
+void main() async {
   WidgetsBinding widgetsBinding = WidgetsFlutterBinding.ensureInitialized();
   FlutterNativeSplash.preserve(widgetsBinding: widgetsBinding);
 
   setupFlutterLogs();
 
+  try {
+    FLog.info(text: "Initialising Firebase");
+    await initFirebase();
```

Comment:
🔧 we should put all the firebase specific code into a service instead of having it in the main.dart

Reply:
moved into a dedicated module.